src/components/PreviewFooter/index.tsx

```diff
@@ -19,16 +26,20 @@ const PreviewFooter = ({ removeAll, multiple, send, sizeLimitExceeded }: Props)
       justifyContent: 'space-between',
       flex: 1,
       paddingBottom: 10,
-    }}
-  >
+    }}>
     <Button
       backgroundColor="transparent"
       onPress={removeAll}
       icon={{ name: multiple ? 'delete-sweep' : 'delete', size: 20 }}
       title={multiple ? 'All' : 'Delete'}
       fontSize={18}
-      buttonStyle={{ height: 42 }}
+      buttonStyle={{ height: 42, backgroundColor: '#ff3030', borderRadius: 10 }}
```

Comment:
It would be better to have color constants in a separate file.

Reply:
well, it would be nice to unify and refactor all the stylings throughout the app, but I considered that as a nice to have (all together with refactoring to hooks)

Reply:
👍